Kempe/Kempe.py

```diff
@@ -76,7 +65,7 @@ def chromatic_polynomial(k, n, m, G, vert):
                 if j not in G2[i]:
                     G2[i].append(j)
     sz = sum(len(G2[i]) for i in vert2) // 2
-    return (chromatic_polynomial(k, n, m - 1, G1, vert) % MOD - chromatic_polynomial(k, n - 1, sz, G2, vert2) % MOD + MOD) % MOD
+    return (chromatic_polynomial(k, n, m - 1, G1, vert) - chromatic_polynomial(k, n - 1, sz, G2, vert2) + MOD) % MOD
```

Comment:
I think the modular subtraction formula is (A-B)%MOD = (A%MOD - B%MOD)%MOD, so no need to remove those `MOD`s, but we can remove the added one.